plugins/services/src/js/structs/__tests__/Pod-test.js

```diff
@@ -211,6 +212,58 @@ describe("Pod", function() {
     });
   });
 
+  describe("#getTaskCount", function() {
+    it("should return 0 if there are no instances", function() {
```

Comment:
Even though lots of our tests have the word "should" prepending the name, we like to avoid it. There's a big open PR removing some https://github.com/dcos/dcos-ui/pull/2524

Reply:
Thank you, will change that 👍